dask/dataframe/utils.py

```diff
@@ -556,7 +556,9 @@ def assert_eq(
     elif isinstance(a, pd.Series):
         a = _maybe_sort(a)
         b = _maybe_sort(b)
-        tm.assert_series_equal(a, b, check_names=check_names, **kwargs)
+        tm.assert_series_equal(
+            a, b, check_names=check_names, check_dtypes=check_dtypes, **kwargs
+        )
```

Comment:
Test failure mentioned above seems to be due to the fact that `check_dtypes` was not forwarded in this case